pkg/autoops/api/progressive_rollout_operation.go

```diff
@@ -161,31 +157,7 @@ func updateFeatureTargeting(
 	return nil
 }
 
-func getResetFeatureCmds(
```

Comment:
I'm removing this implementation because the progressive rollout must only change the default strategy.